client/README.md

```diff
@@ -0,0 +1,447 @@
+
+Each entry in the `clusters` array conforms to a cluster specification (“spec”). A cluster spec requires a name and a url pointing to the swagger spec for a Cook scheduler. Additionally, an optional `http_client` can be provided in order to add custom behavior, like authentication. If the `http_client` type takes the form of `module:function` (note the colon), the module is loaded and the function is called with the provided params. The function should return an `http_client`, and can be used (for instance) to provide custom authentication mechanisms.
+
+### Commands
+
+The fastest way to learn more about `cook` is with the `-h` (or `—help`) option.
+
+```
+usage: cook [-h] [--cluster CLUSTER] [--config CONFIG] [--dry-run] [--no-dry-run]
+                   {create-job,create-group,kill-job,kill-instance,retry-job,await-job,await-instance,query-job,query-instance,query-group}
+                   ...
+
+positional arguments:
+  {create-job,create-group,kill-job,kill-instance,retry-job,await-job,await-instance,query-job,query-instance,query-group}
+    create-job          create job for command
+    create-group        create group
+    kill-job            kill job related to uuid
+    kill-instance       kill instance related to uuid
+    retry-job           retry job related to uuid
+    await-job           await job related to uuid
+    await-instance      await instance related to uuid
+    query-job           query job related to uuid
+    query-instance      query instance related to uuid
+    query-group         query group related to uuid
+
+optional arguments:
+  -h, --help            show this help message and exit
+  --cluster CLUSTER, -c CLUSTER
+                        cluster to use
+  --config CONFIG, -C CONFIG
+                        config file to use
+  --dry-run, -d         print action and arguments without submitting
+  --no-dry-run
+
+```
+
+All global options (`—cluster`, `—config`, `—dry-run`) can be provided when using subcommands. `—dry-run` is useful for testing commands without actually attempting to submit.
+
+In cases where a sub-command can take multiple positional arguments, it’s possible to pass a single dash (`-`) instead. This will cause `cook` to read input from `stdin`, with each line corresponding to one positional argument. In this way, you can pipe the output from one command into the input of another. It also makes it possible to operate on multiple entities at once. For instance, you could start multiple jobs by piping one command per line to `cook create-job`.
```

Comment:
Is it possible to accomplish this without the dash (e.g. `printf 'echo 1' | cook create-job`)?